src/site.rs

```diff
@@ -165,11 +165,13 @@ fn insert_data_dir(data: &mut liquid::Object, data_root: &path::Path) -> Result<
             .file_stem()
             .expect("Files will always return with a stem");
         let file_stem = String::from(file_stem.to_str().unwrap());
-
+        trace!("Loading data from {:?}", &full_path);
         let data_fragment = load_data(&full_path)?;
 
         deep_insert(data, rel_path, file_stem, data_fragment)?;
```

Comment:
Could you also add `chain_err` t both `load_data` and `deep_insert` calls to mention the file, something like

```rust
let data_fragment = load_data(&full_path).chain_err(|| format!("Failed to load data file {:?}", rel_path");
...
deep_insert(data, rel_path, file_stem, data_fragment).chain_err(|| format!("Failed to merging data from {:?}", rel_path");
```

This seems more important than the traces because
- it makes the extra information relevant to the exact use case
- People get it when the failure happens rather than having to do a second run.